RAW/loading.tps

```diff
@@ -0,0 +1,277 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data version="1.0">
+    <struct type="Settings">
+        <key>fileFormatVersion</key>
+        <int>4</int>
+        <key>texturePackerVersion</key>
+        <string>4.4.0</string>
+        <key>fileName</key>
+        <string>D:/Developer/Games/Nya/RAW/loading.tps</string>
```

Comment:
It seems like this is a developer-specific path. Should the file (or folder) be added to `.gitignore`?